api/src/test/java/org/openmrs/api/ProgramWorkflowServiceTest.java

```diff
@@ -127,6 +127,33 @@ public void savePatientProgram_shouldUpdatePatientProgram() {
 		
 	}
 
+	/**
+	 * @see ProgramWorkflowService#voidPatientProgram(PatientProgram, String)
+	 */
+	@Test
+	public  void voidPatientProgram_shouldVoidPatientProgram() {
```

Comment:
please create one test per case since tests should be independent, if one case fails the other test case if its a separate one will not necessarily fail but if you test them in one test this will not be visible and make it harder to fix the problem